os/network-config-with-networkd.md

```diff
@@ -70,8 +70,6 @@ By default, even if you've already set a static IP address and you have a workin
 
 coreos:
   units:
-    - name: systemd-networkd.service
-      command: stop
```

Comment:
Are you sure? The network can sometimes be up before cloudinit runs. I'm assuming this is here for a reason.

cc @crawford